Reescrevendo a documentação da PrestaShop (#1408)
mercadopago/devsite-docs: guides/plugins/prestashop/testing.en.md

# Payment test
 

> The Mercado Pago plugin comes with the **Sandbox teste environment** active by default. In this environment you'll be able to simulate in-store payments and see if everything is working correctly before you start receiving real payments from your customers. To perform the test, follow the steps below.

Reviewer: ```suggestion
The Mercado Pago plugin comes with the **Sandbox test environment** active by default. In this environment you'll be able to simulate in-store payments and see if everything is working correctly before you start receiving real payments from your customers. To perform the test, follow the steps below.
```

> 1. On your store's Administrative Panel, access the Modules and Services menu, locate the Mercado Pago plugin and click **configure**.

Reviewer: ```suggestion
1. On your store's Administrative Panel, access the Modules and Services menu, locate the Mercado Pago plugin and click on **configure**.
```

> 2. On the plugin management screen, confirm that the test [credentials](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/resources/credentials) are properly filled in accordingly with the information available on its [Dashboard](https://www.mercadopago.com.br/developers/panel).

Reviewer: creo que el link del dashboard deberia ser un faker url

3. Then scroll down to the **Test Your Store** section and click on **I want to test my sales**.
 

> ![Testing payments](/images/prestashop/teste_pagto_en.png)

Reviewer: la imagen no se ve porque no existe en el proyecto

> 4. On your PrestaShop store home page, select a product and click to add in your cart.

Reviewer: ```suggestion
4. On your PrestaShop store home page, select a product and click to add to your cart.
```

5. With the product selected, complete the order.

> 6. Fill in with personal information, indicate a shipping address and select a shipping method. Remembering that all this information is just for testing.

Reviewer: ```suggestion
6. Fill in with personal information, indicate a shipping address and select a shipping method. Remember that all this information is just for testing.
```

7. Select the payment option with the Mercado Pago solution you configured.

> 8. Use the [test credit cards](https://www.mercadopago.[FAKER][URL][DOMAIN]/developers/en/guides/resources/localization/local-cards) to make the payment.

Reviewer: ```suggestion
8. Use the [test credit cards](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/resources/localization/local-cards) to make the payment.
```

> 9. When completing the filling, **finalize the purchase**.

Reviewer: ```suggestion
9. When you've completed everything, click on **finalize the purchase**.
```

> After completing the previous steps, activate **Production mode** to receive actual payments.

Reviewer: ```suggestion
After completing the previous steps, activate **Production mode** to receive real payments.
```

 
> LEFT_BUTTON_REQUIRED_EN
>
> Receiving payments
>

> > Learn how to enable your store to process actual sales.

Reviewer: ```suggestion
> Learn how to enable your store to process real sales.
```
